### PR TITLE
Radam support the flag for "maximize"

### PR DESCRIPTION
**Pull request by huihoaan**
Fixes #[126642](https://github.com/pytorch/pytorch/issues/126642)

I reference the maximize in `Adam` and add `Radam's` maximize flag. If this pr is OK, I will add another pr for `Nadam`.

### CODE REVIEW

File: torch/optim/radam.py [hunk 1]
```diff
@@ -169,14 +174,17 @@ def step(self, closure=None):
...
             &\textbf{initialize} :  m_0 \leftarrow 0 \text{ ( first moment)},
                 v_0 \leftarrow 0 \text{ ( second moment)},                                       \\
             &\hspace{18mm} \rho_{\infty} \leftarrow 2/(1-\beta_2) -1                      \\[-1.ex]
             &\rule{110mm}{0.4pt}  \\
             &\textbf{for} \: t=1 \: \textbf{to} \: \ldots \: \textbf{do}                         \\
-            &\hspace{6mm} g_t \leftarrow \nabla_{\theta} f_t (\theta_{t-1})                      \\
+            &\hspace{6mm}\textbf{if} \: \textit{maximize}:                                       \\
+            &\hspace{12mm}g_t           \leftarrow   -\nabla_{\theta} f_t (\theta_{t-1})         \\
+            &\hspace{6mm}\textbf{else}                                                           \\
+            &\hspace{12mm}g_t           \leftarrow   \nabla_{\theta} f_t (\theta_{t-1})          \\
```
Comment (janeyx99) [inline torch/optim/radam.py:187]: can you link a screenshot to the doc preview to ensure this rendered correctly?

**Review by janeyx99 — APPROVED** [review]
This is very good! Thank you for the speed!

**Comment by huihoaan** [comment]
@janeyx99  the doc is as follow:
image

**Comment by huihoaan** [comment]
@janeyx99  I have make it default kwarg for `radam`， can you take a look again？

**Pull request closed without merging**